packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -965,6 +965,10 @@
   "Markdown_Headers": "Allow Markdown headers in messages",
   "Markdown_SupportSchemesForLink": "Markdown Support Schemes for Link",
   "Markdown_SupportSchemesForLink_Description": "Comma-separated list of allowed schemes",
+  "Markdown_ColorSupport_Description": "Markdown Support for Colors",
```

Comment:
Can you describe the difference between the two options in this description? That way users know what is supported and not supported by the description of the setting and don't have to go looking through pages and pages to find out what it means. Thanks!